### PR TITLE
Table style tweaks

### PR DESCRIPTION
**Pull request by zarsky-broad**
Made w/ Lou. @breilly2 how is this for your eyes?

### CODE REVIEW

File: src/components/table.js [hunk 1]
```diff
@@ -108,29 +108,25 @@ export const paginator = props => {
...
+})
 
 const styles = {
   cell: (col, total) => ({
-    ...cellStyles,
+    ...cellStyles(col, total),
     borderBottom: `1px solid ${colors.dark(0.2)}`,
-    borderLeft: `1px solid ${colors.dark(0.2)}`,
-    borderRight: col === total - 1 ? `1px solid ${colors.dark(0.2)}` : undefined
+    borderLeft: col === 0 ? undefined : `1px solid ${colors.dark(0.2)}`
```
Comment (panentheos) [inline src/components/table.js:122]: TOL: For me, this has a couple impacts: On the pages where the table butts up against the edge of the screen (data, jobs), it makes me think there's content off screen, and I keep wanting to scroll to it. And in the places where it doesn't (modal, method config), it looks like a style bug. Thoughts?

**Review by panentheos — APPROVED** [review]
Code looks fine, see comment.

File: src/components/table.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/table.js:122]: Yeah, I definitely think this is worth discussing with the team and Lou. I'm thinking of this PR as a design tweak proposal.

Comment (breilly2) [reply src/components/table.js:122]: I agree with Brett. There are definitely areas where it needs work. To me, expanding tables like this shifts the paradigm from an application with scrollable panes to a scrollable document with floating tool/navigation areas.

**Comment by panentheos** [comment]
This change needs more thinking/discussion, and this PR is getting stale, so we're going to close it for now.

**Pull request closed without merging**